objects/rct2tt/ride/rct2.tt.dragnfly.json

```
@@ -53,6 +53,7 @@
             "sv-SE": "Trollslända",
             "ko-KR": "잠자리 차량",
             "zh-TW": "大蜻蜓雲霄飛車",
+            "ja-JP": "蜻蛉車両",
```

This word is usually written in hiragana. Let's go with:
```suggestion
            "ja-JP": "とんぼ車両",
```